POH-README.md

```diff
@@ -14,8 +14,15 @@ program and the data will be kept.  POH uses its own version of the Java Virtual
 Machine and the MySQL server and everything is contained in the root
 installation directory.  POH is released under the GNU GPL 3.0 License.
 
-The Linux version has been tested on Ubuntu 16.04 i386 and on Arch Linux x64
-updated to July 2019. The Windows version has been tested on Windows 10.  
+The Linux version has been tested on Ubuntu 20.04 x64 updated to October 2020.

```

Comment:
Not sure why you removed the mention to Ubuntu 16.04 i386. It's actually good to say that we tested the x32 distribution on the latest x32 Ubuntu [...if we decide to still ship the x32 Linux portable - IMHO we can stop shipping it].

Reply:
I agree with you, sorry about that, I missed the i386 part so I thought about to just mention the latest release. Corrected.

BTW, I also agree that we will have to distribute only the 64bit in the very near future: Linux 32bit system are rare now _-)

Reply:
Maybe it would be hard to mention *all* systems on which has been tested... so we could just not mention it or most likely just mention the latest?

We still have issues on Win64 as per OP-314, and OP-228 is still ongoing for other platform/architectures. I think first we complete the assessment, then we decide what to ship.

Reply:
I thinkl we could just add something like "Latest release tested was Ubuntu xx.xx".

Reply:
Updated README.

Reply:
Updated. I think it should be ok now.